src/sentry/static/sentry/app/components/forms/checkboxField.jsx

```diff
@@ -47,7 +47,7 @@ export default class CheckboxField extends FormField {
           <input type="checkbox"
                  onChange={this.onChange.bind(this)}
                  disabled={this.props.disabled}
-                 selected={this.props.value} />
+                 checked={this.props.value} />
```

Comment:
lol this somehow never worked because the correct field is `checked`, not `selected`.